scheduler/forms.py

```diff
@@ -1,7 +1,10 @@
 """Forms"""
+
 from django import forms
-from bootstrap_modal_forms.forms import BSModalForm
```

Comment:
Może przy okazji pozbyć się tego z requirements.txt: django-bootstrap-modal-forms==1.4.2? Chyba nie jest to więcej używane

Reply:
I jeszcze linijki 5-8 z index.html - bo chyba nic nie dają oprócz logów 404 na konsoli

Reply:
A po usunięciu tego to w ogóle mi się formularze nie wyświetlają, więc brzmi jakby jeszcze było potrzebne

Reply:
Serio? U mnie działa

Reply:
Usunąłem tyle żeby jeszcze działało